src/svg/ds4/icons.svg

```diff
@@ -231,6 +231,9 @@
     </symbol>
     <symbol id="icon-priority" viewBox="0 0 24 24">
         <path fill="#dd1e31" d="M12.007 14.261a.849.849 0 0 1-.849-.849V6.859a.849.849 0 1 1 1.684.005v6.549a.849.849 0 0 1-.834.849l-.001-.001zm1.057 2.894a1.016 1.016 0 1 1-2.032 0 1.016 1.016 0 0 1 2.032 0zm-1.057 6.859C5.376 24.014 0 18.638 0 12.007 0 5.376 5.376 0 12.007 0c6.631 0 12.007 5.376 12.007 12.007v.015c-.008 6.625-5.381 11.992-12.007 11.992zm0-22.261c-5.671 0-10.268 4.597-10.268 10.268s4.597 10.268 10.268 10.268 10.268-4.597 10.268-10.268S17.678 1.753 12.007 1.753z"/>
+    </symbol>
+        <symbol id="icon-attention" viewBox="0 0 24 24">
```

Comment:
Ultra-super nit-picky: needs one less indentation.

Reply:
How dare you, @sgates! Haha, kidding, lemme change this.

Reply:
fixed indentation & added deprecation comments